schema.sql

```diff
@@ -65,3 +64,25 @@ CREATE TABLE visits (
     CONSTRAINT unique_visit_animal_vet UNIQUE (animal_id, vet_id, visit_date)
 );
 
+ALTER TABLE visits ADD COLUMN date_of_visit timestamp;
+
+SELECT COUNT(*) FROM visits WHERE animal_id = 4;
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE animal_id = 4;
+
+
+SELECT COUNT(*) FROM visits where animal_id = 4;
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE animal_id = 4;
+
+
+SELECT * FROM visits where vet_id = 2;
+SELECT * FROM owners where email = 'owner_18327@mail.com';
+
+CREATE INDEX idx_animal_id ON visits(animal_id);
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE animal_id = 4;
+
+CREATE INDEX idx_vet_id ON visits(vet_id);
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE vet_id = 4;
+
+
+CREATE INDEX idx_email ON owners(email);
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits WHERE email = 4;
```

Comment:
- You have done great so far :+1: :+1:  but there are a few requirements and changes you need to make before moving to the next milestone. :wave: 

- [ ] Currently, looking at the screenshot in your pull request description, it is a bit difficult to see the results of the queries before the database improvements and screenshots after the improvements. I'll suggest you take the screenshots individually and place them accordingly to make things easier, Looking at the screenshot below, you can see how its arranged to show before and after improvement for each query. :+1: :+1: 

- - Screenshot reference:
![Selection_005](https://github.com/Simpleshaikh1/vetclinic-db/assets/107632780/f73ba4ae-0d08-4f75-ba89-57a54e6c2a6e)